docs/content/installation/build-nginx-ingress-controller.md

```diff
@@ -192,7 +194,7 @@ If you prefer not to build your own NGINX Ingress Controller image, you can use
 
 **NGINX Ingress Controller**: Download the image `nginx/nginx-ingress` from [DockerHub](https://hub.docker.com/r/nginx/nginx-ingress) or [GitHub](https://github.com/nginxinc/kubernetes-ingress/pkgs/container/kubernetes-ingress).
 
-**NGINX Plus Ingress Controller**: You have two options for this, both requiring an NGINX Ingress Controller subscription.
+**NGINX Plus Ingress Controller**: You have two options for this: both require an NGINX Ingress Controller subscription.
```

Comment:
For the line `You have two options for this: both require an NGINX Ingress Controller subscription.` I wonder if we even need to have the part where it mentions needing a subscription? Since we are in the `NGINX Plus Ingress Controller` section, having a subscription would be a given? What do you think?

If we did we remove that part, this section could look like this:

```suggestion
**NGINX Plus Ingress Controller**: You have two options for this:
```